.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -1,117 +1,98 @@
 /*
```

Comment:
@marcosgopen this commit when read in conjunction with https://issues.redhat.com/browse/JBTM-3783 is not clear why the upgrade to Maven is required. Please can you add a note into the JBTM to why it's required?

Reply:
Thanks Tom for your comment. I added a comment on the issue, I forgot to add a link from Zulip thread ["Minimal mvn version"](https://narayana.zulipchat.com/#narrow/stream/323715-developers/topic/.E2.9C.94.20Minimal.20mvn.20version) where we discuss about it.